### ISSUE TITLE
uninitialized constant Flipper::VERSION with 0.27 bump

### ISSUE THREAD

**Issue by brendo** [jnunemaker/flipper#711]
After the Flipper 0.27 bump, we are seeing the following error when trying to access the Admin UI.

Ruby 3.2.1, Rails 7.0.4.3

```
(eval):23:in `view': uninitialized constant Flipper::VERSION (NameError)
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:248:in `eval'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:248:in `view'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:235:in `view_with_layout'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:165:in `view_response'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/actions/features.rb:35:in `get'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:127:in `block in run'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:127:in `catch'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:127:in `run'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/action.rb:86:in `run'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/middleware.rb:51:in `call!'
    from gems/flipper-ui-0.27.0/lib/flipper/ui/middleware.rb:40:in `call'
    from gems/flipper-0.27.0/lib/flipper/middleware/setup_env.rb:45:in `call!'
    from gems/flipper-0.27.0/lib/flipper/middleware/setup_env.rb:40:in `call'
    from gems/rack-2.2.6.4/lib/rack/method_override.rb:24:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/xss_header.rb:20:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/base.rb:53:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/base.rb:53:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/path_traversal.rb:18:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/json_csrf.rb:28:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/base.rb:53:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/base.rb:53:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/frame_options.rb:33:in `call'
    from gems/rack-protection-3.0.5/lib/rack/protection/base.rb:53:in `call'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/routing/mapper.rb:19:in `block in <class:Constraints>'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/routing/mapper.rb:48:in `serve'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/journey/router.rb:50:in `block in serve'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/journey/router.rb:32:in `each'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/journey/router.rb:32:in `serve'
    from gems/actionpack-7.0.4.3/lib/action_dispatch/routing/route_set.rb:852:in `call'
    from gems/flipper-0.27.0/lib/flipper/middleware/memoizer.rb:72:in `memoized_call'
    from gems/flipper-0.27.0/lib/flipper/middleware/memoizer.rb:37:in `call'
    from gems/rack-2.2.6.4/lib/rack/tempfile_reaper.rb:15:in `call'
    from gems/rack-2.2.6.4/lib/rack/etag.rb:27:in `call'
    from gems/rack-2.2.6.4/lib/rack/conditional_get.rb:27:in `call'
    from gems/rack-2.2.6.4/lib/rack/head.rb:12:in `call'
    ...     
```

**Comment by larouxn** [comment]
Looks like that call was added [here](https://github.com/jnunemaker/flipper/pull/709/files#diff-283cc728fe0d3d9dd980e7955440bfb4aa2a1aa490abe7bd41d37e107c3ad140R23).

https://github.com/jnunemaker/flipper/blob/86f1be6d4e4cf2dbfaabbe78fbb96717bae5d57f/lib/flipper/ui/views/layout.erb#L20-L24

**Comment by joker-777** [comment]
I'm haveing the same issue here.

**Comment by jnunemaker** [comment]
All fixed up in 0.27.1. Sorry folks!
